towhee/base_repr.py

```diff
@@ -25,30 +22,26 @@ class BaseRepr:
         name:
             Name of the internal object described by this representation.
     """
-
-    def __init__(self, name: str):
-        self._name = name
+    def __init__(self, name: str = None):
+        if name:
```

Comment:
if the name is None, just set self._name = None，make sure this attr is exist

Reply:
fixed